lib/dotcom_web/live/subway_alerts.ex

```diff
@@ -0,0 +1,115 @@
+defmodule DotcomWeb.Live.SubwayAlerts do
+  @moduledoc """
+  The subway alerts/status page, showing current subway status, planned work, and
+  other upcoming and relevant alerts.
+  """
+
+  use DotcomWeb, :live_view
+
+  import Dotcom.Alerts, only: [service_impacting_alert?: 1]
+  import DotcomWeb.Components.PlannedDisruptions, only: [disruptions: 1]
+  import DotcomWeb.Components.SystemStatus.SubwayStatus, only: [alerts_subway_status: 1]
+
+  alias Alerts.InformedEntity
+  alias Alerts.Match
+  alias Dotcom.Alerts.Subway.Disruptions
+  alias Dotcom.SystemStatus
+
+  @alerts_repo Application.compile_env!(:dotcom, :repo_modules)[:alerts]
+  @routes_repo Application.compile_env!(:dotcom, :repo_modules)[:routes]
+
+  @date_time Application.compile_env!(:dotcom, :date_time_module)
+
+  embed_templates "layouts/*"
+
+  def mount(_params, _session, socket) do
+    {:ok,
+     socket
+     |> assign(
+       :meta_description,
+       "Live service alerts for all MBTA transportation modes, including subway, bus, Commuter Rail, and ferry. " <>
```

Comment:
don't forget to translate this too!

Reply:
[Done](https://github.com/mbta/dotcom/pull/2663/commits/1e0cc0631f775ad915bc2f425059f4b7da1c26bf)